pkg/network/etw/etw-http-service.go

```diff
@@ -641,6 +675,7 @@ func httpCallbackOnHTTPRequestTraceTaskParse(eventInfo *C.DD_ETW_EVENT_INFO) {
 		// in future it probably should be optimize because we have already
 		// whole thing
 		httpConnLink.http.RequestFragment = make([]byte, maxRequestFragmentBytes)
+		httpConnLink.http.Txn.MaxRequestFragment = uint16(maxRequestFragmentBytes)
```

Comment:
So there's a couple of things here that need to be unpacked.
The intention of this was to have it be the actual length so that only the actual length was copied if the fragment was less than the max.
Also, the driver is runtime configurable so it could, in theory, change.